src/main/assembly/dist/extern/springfield-actions.xsd

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<xs:schema xmlns:sf="springfield-actions" xmlns:xml="http://www.w3.org/XML/1998/namespace"
-           targetNamespace="springfield-actions" xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified">
+<xs:schema xmlns:sf="http://easy.dans.knaw.nl/extern/springfield-actions/" xmlns:xml="http://www.w3.org/XML/1998/namespace"
```

Comment:
Shouldn't that be "external"?

Reply:
maybe that's better. i just copied the word from the `easy-schema` URL

Reply:
in that case, I probably merged https://github.com/DANS-KNAW/easy-ingest-flow/pull/108 too quickly 🤔

Reply:
Okay let us know if it becomes external, as we need to change it in easy-ingest-flow as well, as Richard indicated.

Reply:
@rvanheest : i've changed it from 'extern' to 'external'

Reply:
I've also changed it in https://github.com/DANS-KNAW/easy-ingest-flow/pull/111